GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -87,6 +87,11 @@ public void Execute()
                 // any platform specific preparation required to run as a background process.
                 GVFSPlatform.Instance.PrepareProcessToRunInBackground();
             }
+
+            if (GVFSPlatform.Instance.FileSystem.IsCaseSensitive(this.EnlistmentRootPathParameter))

```

Comment:
**This will block the Linux team.**

Instead: maybe make this an `IsFileSystemSupported()` method on the platform, and the Mac platform rejects case-sensitive filesystems?

Reply:
@derrickstolee updated to IsFileSystemSupported